test/test_utils.js

```diff
@@ -81,3 +82,50 @@ export function sessionFactory(props) {
     redirectURL: "",
   };
 }
+
+// temporary solution until we rewrite AdminLink to not require router around it
```

Comment:
Do we have an issue to link here for reference?

Reply:
I've changed my mindset on this one a bit. Mocking Link to avoid issues with react-router is easy and I'm probably going to do that in focused unit tests going forward, so no need to remove it. 

However because we're very tied to redux right now we can't get away from Provider as easily, so there is less of a point on larger unit tests. So I created the `renderWithProvider` function that we can start migrating towards. I'm hoping this causes less friction in short and long term changes with our unit tests in this repo.